azure-pipelines.yml

```diff
@@ -0,0 +1,38 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+pr:
+- master
+
+trigger: none
+
+pool:
+  vmImage: 'Ubuntu-16.04'
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '8.x'
+  displayName: 'Install Node.js'
+
+- script: |
+    npm install
+  displayName: 'Install npm dependencies'
+
+- script: |
+    npm audit
```

Comment:
Per the `lodash` incident and `npm audit` not respecting the `--production` flag, are we removing it?